src/base1/cockpit.js

```diff
@@ -841,6 +841,7 @@ function Channel(options) {
         else
             delete command.binary;
 
+        command["flow-control"] = true;
```

Comment:
Just to understand -- the *binary* cockpit-bridge and thus the protocol default to "off" -- but the JavaScript API defaults to on, so that as soon as one uses a newer cockpit-system they get flow control enabled for everything. This would still work with new cockpit-system  and old cockpit-bridge as unknown options get ignored. Did I understand that correctly?

Reply:
Yes, that's correct.